dts/arc/quark_se_c1000_ss.dtsi

```diff
@@ -0,0 +1,50 @@
+#include "skeleton.dtsi"
+#include "mem.h"
+
+/ {
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu@0 {
+			device_type = "cpu";
+			compatible = "arc";
```

Comment:
take a look at the linux dts for arc, guessing this should be something like "snps,arcXXXXX";

Reply:
ok.